services/gateway/apis/http-version3/swagger/parameters/common.json

```
@@ -1,8 +1,25 @@
 {
+    "addressCsv": {
+        "name": "address",
+        "in": "query",
+        "description": "Address of the account to query (can be expressed as CSV).",
+        "type": "string",
+        "format": "address",
+        "minLength": 41
```

May be we can also add maxLength:
```suggestion
        "minLength": 41,
        "maxLength": 419
```

---

This condition may not be true for other places where we may use this description in future. will skip for this reason.